ScipDotnet/Program.cs

```diff
@@ -42,7 +42,11 @@ public static async Task<int> Main(string[] args)
                 "If enabled, allow public symbol definitions to be accessible from other SCIP indexes. " +
                 "If disabled, then public symbols will only be visible within the index."),
             new Option<int>("--dotnet-restore-timeout", () => DotnetRestoreTimeout,
-                @"The timeout (in ms) for the ""dotnet restore"" command")
+                @"The timeout (in ms) for the ""dotnet restore"" command"),
+            new Option<bool>("--dotnet-skip-restore", () => false,
+                @"Skip executing ""dotnet restore"" and assume it has been run externally."),

```

Comment:
nit: I think "skip-dotnet-restore" (matching the explanatory comment) would be a better fit.

Reply:
I went back and forth but decide to keep all the commands that were specific for targeting how we ran the underlying dotnet cli operations prefixed with dotnet.

Reply:
But given your other comment, not sure its helpful, so will change